src/models/inverter_models/filter_models.jl

```diff
@@ -79,12 +79,12 @@ function mdl_filter_ode!(
     #Grid Inductance (internal state)
     #𝜕id_o/𝜕t
     output_ode[local_ix[5]] = (
-        ωb / lg * Vd_filter - ωb / lg * V_dq[2] - ωb * rg / lg * Id_filter +
+        ωb / lg * Vd_filter - ωb / lg * V_dq[1] - ωb * rg / lg * Id_filter +
         ωb * ω_sys * Iq_filter
     )
-    #𝜕iq_o/𝜕t
+    #𝜕iq_o/𝜕t (Multiply Vq by -1 to lag instead of lead)
```

Comment:
it might take a while to disentangle in the inverter model the correct `_d` and `_q` designation to make the reference frames consistent.